### DISCUSSION TITLE
"Convert to issue" feature requests

### DISCUSSION THREAD

**Discussion by tallclair** [community/community#4330] [Projects and Issues]
I love the idea of converting a task list item to an issue, but the current implementation is a bit too sparse to be very useful to me. I would like to see the converted issues carry over more data from the parent issue, including:

- Milestone
- Projects
- Labels (I could see how one might not want all labels to copy over)
- ** Subtasks: if I have a nested task list (or nested bullets without tasks), converting the parent should copy the nested bullets into the converted issue.

**Comment by bomeejung** [comment]
+1 on this request/issue!
At the very minimum, the subtask not inheriting the Projects from the parent means it doesn't show up at all in the Project board

**Comment by qwinters** [comment]
+1 and also worth mentioning that this affects how the milestone percentage appears (I think it is more accurate to add the tasks to the milestone which might make the milestone look less complete but reflects that the original issue is harder than previously imagined)

**Comment by nexushoratio** [comment]
Some possible partial duplicates

- #4330 
- #12172

**Comment by e12e** [comment]
There should perhaps be an api hook on issue identifying it's origin as converted from subtask on parent issue? For example we would like to send "new" (manually created) issues to "epic" and subtask issues to "backlog".

As far as I can tell, there's no way to see that a newly created issue originates as a subtask right now, so that we could implement this flow as an action?

**Comment by e12e** [comment]
Apparenlty there's a beta concept of "tracking issues", but a) Doesn't appear to be exposed in the API? And b) I don't see these fields listed under hidden views in my project - nor under features to add as "Feature preview".

https://docs.github.com/en/issues/planning-and-tracking-with-projects/understanding-fields/about-tracks-and-tracked-by-fields

https://docs.github.com/en/get-started/using-github/exploring-early-access-releases-with-feature-preview
